### PR TITLE
Add switch plugin to krew index

### PR DESCRIPTION
**Pull request by radraw**

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

**Comment by ahmetb** [comment]
The name is too generic and therefore we cannot assign. It can refer to many things. Please refer to Naming guidelines.

**Comment by radraw** [comment]
> The name is too generic and therefore we cannot assign. It can refer to many things. Please refer to Naming guidelines.

I've read guidelines. There are already many plugins with generic names. From practical point of view and intuition I think switching of clusters is nearest assotiation. But this may be just my view. Consider please bending a little bit naming guidelines for this case, maybe it's worth. For shortness and simplicity. If you still insist on name change, I can do it. Perhaps I will have to provide some kind of alias for my current  (local) users of this plugin or communicate that it's name changed.

**Comment by ahmetb** [comment]
With all due respect when you say "switch" in Kubernetes context, I think of kubectx/kubens and we didn't assign those plugins that name. Switching kubeconfig files didn't even ring a bell for me. 

Many plugins distribute with a slightly different name in Krew and that's usually ok for users.

We try very hard not to bend the rules. We've already had many cases where different plugins were submitted with the same names over time and we avoid this firstcomer advantage.

/kind gray-area

**Comment by radraw** [comment]
Now the code should be visible as I've made it public. Sorry I though I made it before, but apparently not.

Regarding name, I think "switch-config" should be conforming to the naming rules. Let me know if it's ok for you and I will make the change.

**Comment by radraw** [comment]
As I see the PR is waiting to satisfy check "tide Pending — Not mergeable. Needs approved, lgtm labels.". I don't quite know what that is, and I don't think it's on my side, but just to be sure.. Is there something missing from my side to finish this PR ?

File: plugins/switch-config.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: switch-config
+spec:
+  version: "v0.1.0"
+  homepage: https://github.com/radraw/kube-switch-config
+  shortDescription: "Switches between Kubernetes configuration files"
+  description: |
+    Simple switching between configuration files using symbolic links.
```
Comment (ahmetb) [inline plugins/switch-config.yaml:10]: the terms `configuration files` and `Kubernetes configuration files` don't mean much to kubernetes users (feel free to google them, you won't see them being referred to the same thing).

I'd encourage you to use `kubeconfig file` as the term here for consistency with the rest of the ecosystem.

**Comment by ahmetb** [comment]
/lgtm
/approve
Thanks!

**Pull request merged**